src/optimalcontrol/src/ConstraintsGroup.cpp

```diff
@@ -39,6 +45,13 @@ namespace optimalcontrol {
             std::vector< TimedConstraint_ptr > orderedIntervals;
             std::string name;
             unsigned int maxConstraintSize;
+            std::vector<TimeRange> timeRanges;
+
+            std::vector< TimedConstraint_ptr >::reverse_iterator findActiveConstraint(double time){
```

Comment:
still trying to add space between the angular brackets and the template name? 🙄

What does `findActiveConstraint` do? Is it possible to iterate on the resulting iterator?

Reply:
I am trying to lose this habit :smile:. 

I was needing a pointer to the last ``TimedConstraint_ptr`` whose ``TimeRange`` includes ``time``, i.e. the constraint which should be activated given that time among all the constraints in a specific group.